config/proto/service.proto

```diff
@@ -1,9 +1,22 @@
-/*
-  This proto defines the service-level config for Hermes.
-  This is also the external service API for Hermes.
-  
-  Author: Evan Spendlove (@evanSpendlove)
-*/
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author: Evan Spendlove (@evanSpendlove)
+//
+//  This proto defines the service-level config for Hermes.
```

Comment:
Same as above: this should be a repeated field.

Reply:
I am addressing comments on the .proto files in a different PR, as this branch is meant to only include the Cloudprober integration files. These .proto files are the same as those in the master branch.

Please see the new PR for the branch: evan/protoChanges.

--

I have made this a repeated field with the appropriate naming.